src/Components/CommentSection.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {FormEvent, useState} from "react";
+import CommentDialogSection from "./CommentDialogSection";
+import {addComment, getComments} from "../Service/User/RequestsForUsers";
+import {useAppSelector} from "../Hooks/hook";
+import {commentSchema} from "../validator";
+import {ToastNotifyError, ToastNotifySuccess} from "../Helpers";
+import {CommentFromDB, User} from "../types/typeSection";
+import CoreButton from "./common/CoreButton";
+
+interface Props {
+    postId: number
+}
+
+const CommentSection: React.FC<Props> = ({postId}) => {
+    const user:User|null = useAppSelector(state => state.auth.auth)
+    const [loading,setLoading] = useState<boolean>(false)
+    const [open, setOpen] = useState<boolean>(false);
```

Comment:
spacing is not kept, first row not having space after `loading,` second one having space, integrate prettier and eslint for keeping everything solid.